noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -151,6 +173,24 @@ impl PrivateContext {
         get_header_at(block_number, self)
     }
 
+    pub fn nullifier_valid_inclusion_at(
```

Comment:
If you want to be as consistent as possible within the two it seems better to not have the `_at` in here but specifically use the library for that. Similar for anything but the `exists`?